### PR TITLE
Minimal doc improve

### PR DESCRIPTION
**Pull request by frheault**
Some of the scripts had the wrong argparse formatter and an overly short description. A small attempt to fix that problem.

autopep8 was run on each for indentation and some line length was modified. Complete switch of DESCRIPTION -> __doc__

### CODE REVIEW

File: scripts/scil_visualize_seeds.py [hunk 1]
```diff
@@ -2,8 +2,10 @@
 # -*- coding: utf-8 -*-
 
 """
-Visualize seeds where streamlines originate from
-in a tractogram
+Visualize seeds where streamlines originate from in a tractogram.
```
Comment (arnaudbore) [inline scripts/scil_visualize_seeds.py:5]: from a tractogram.

File: scripts/scil_bundle_mean_std.py [hunk 2]
```diff
@@ -23,7 +27,7 @@
 def _build_arg_parser():
     p = argparse.ArgumentParser(
         description=__doc__,
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        formatter_class=argparse.RawTextHelpFormatter)
```
Comment (arnaudbore) [inline scripts/scil_bundle_mean_std.py:30]: in_metrics

File: scripts/scil_compute_local_tracking.py [hunk 3]
```diff
@@ -44,17 +43,17 @@
...
+        formatter_class=argparse.RawTextHelpFormatter)
 
     p._optionals.title = 'Generic options'
     p.add_argument('sh_file',
                    help='Spherical harmonic file. \n'
-                   '(isotropic resolution, nifti, see --basis).')
+                        '(isotropic resolution, nifti, see --basis).')
     p.add_argument('seed_file',
                    help='Seeding mask (isotropic resolution, nifti).')
     p.add_argument('mask_file',
```
[the remark this answers is not in the document]
Comment (frheault) [reply scripts/scil_compute_local_tracking.py:54]: None of them follow the convention, I would keep that for another PR to keep this one simple (without major testing)

File: scripts/scil_compute_local_tracking.py [hunk 4]
```diff
@@ -44,17 +43,17 @@
...
                    help='Seeding mask(isotropic resolution, nifti).\n' +
-                   'Tracking will stop outside this mask.')
+                        'Tracking will stop outside this mask.')
     p.add_argument('output_file',
```
[the remark this answers is not in the document]
Comment (frheault) [reply scripts/scil_compute_local_tracking.py:57]: Same

File: scripts/scil_compute_local_tracking.py [hunk 5]
```diff
@@ -44,17 +43,17 @@
 def _build_arg_parser():
     p = argparse.ArgumentParser(
         description=__doc__,
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        formatter_class=argparse.RawTextHelpFormatter)
 
     p._optionals.title = 'Generic options'
     p.add_argument('sh_file',
```
[the remark this answers is not in the document]
Comment (frheault) [reply scripts/scil_compute_local_tracking.py:49]: Same

File: scripts/scil_compute_local_tracking.py [hunk 6]
```diff
@@ -44,17 +43,17 @@
...
                    help='Spherical harmonic file. \n'
-                   '(isotropic resolution, nifti, see --basis).')
+                        '(isotropic resolution, nifti, see --basis).')
     p.add_argument('seed_file',
```
[the remark this answers is not in the document]
Comment (frheault) [reply scripts/scil_compute_local_tracking.py:52]: Same

File: scripts/scil_bundle_volume.py [hunk 7]
```diff
@@ -2,7 +2,11 @@
 # -*- coding: utf-8 -*-
 
 """
-Compute bundle volume in mm³.
+Compute bundle volume in mm³. This script support anisotropic voxel resolution.
```
Comment (mdesco) [inline scripts/scil_bundle_volume.py:5]: supports

File: scripts/scil_bundle_volume.py [hunk 8]
```diff
@@ -2,7 +2,11 @@
...
+Volume is estimated by counting the number of voxel occupied by the bundle and
```
Comment (mdesco) [inline scripts/scil_bundle_volume.py:6]: voxels

File: scripts/scil_bundle_volume.py [hunk 9]
```diff
@@ -2,7 +2,11 @@
...
+multiplying it by the volume of a single voxel.
+
+This estimation is typically perform at resolution around 1mm³.
```
Comment (mdesco) [inline scripts/scil_bundle_volume.py:9]: performed

File: scripts/scil_bundle_volume_per_label.py [hunk 10]
```diff
@@ -2,7 +2,11 @@
 # -*- coding: utf-8 -*-
 
 """
-Compute bundle volume per label.
+Compute bundle volume per label in mm³. This script support anisotropic voxel
```
Comment (mdesco) [inline scripts/scil_bundle_volume_per_label.py:5]: supports

File: scripts/scil_bundle_volume_per_label.py [hunk 11]
```diff
@@ -2,7 +2,11 @@
...
+resolution. Volume is estimated by counting the number of voxel occupied by
```
Comment (mdesco) [inline scripts/scil_bundle_volume_per_label.py:6]: voxels

File: scripts/scil_bundle_volume_per_label.py [hunk 12]
```diff
@@ -2,7 +2,11 @@
...
+each label and multiplying it by the volume of a single voxel.
+
+This estimation is typically perform at resolution around 1mm³.
```
Comment (mdesco) [inline scripts/scil_bundle_volume_per_label.py:9]: performed

File: scripts/scil_bundle_voxel_label_map.py [hunk 13]
```diff
@@ -2,8 +2,10 @@
 # -*- coding: utf-8 -*-
 
 """
-Compute (upsampled) Nifti label image from bundle and centroid.
+Compute label image (Nifti) from bundle and centroid.
 Each voxel will have the label of its nearest centroid point.
+
+The number of label will be the same as the centroid's number of point.
```
Comment (mdesco) [inline scripts/scil_bundle_voxel_label_map.py:8]: labels

File: scripts/scil_compute_fodf_max_in_ventricles.py [hunk 14]
```diff
@@ -2,7 +2,10 @@
 # -*- coding: utf-8 -*-
 
 """
-Script to compute the maximum fODF in the ventricles.
+Script to compute the maximum fODF in the ventricles. The ventricules are
+estimated from a MD and FA threshold.
+
+This allows to clip the noise of fODF using an absolute thresold.
```
Comment (mdesco) [inline scripts/scil_compute_fodf_max_in_ventricles.py:8]: We should cite: 
Dell’acqua, F., Simmons, A., Williams, S. C. R., & Catani, M. (2012). Can spherical deconvolution provide more information than fiber orientations? Hindrance modulated orientational anisotropy, a true-tract specific index to characterize white matter diffusion. Human Brain Mapping.

File: scripts/scil_endpoints_map.py [hunk 15]
```diff
@@ -2,13 +2,15 @@
...
 
 """
-Computes the endpoint map of a bundle. The endpoint map
-is simply a count of the number of streamlines that
-start or end in each voxel. The idea is to estimate the
-cortical areas affected by the bundle (assuming
+Computes the endpoint map of a bundle. The endpoint map is simply a count of
+the number of streamlines that start or end in each voxel.
+
+The idea is to estimate the cortical areas affected by the bundle (assuming
```
Comment (mdesco) [inline scripts/scil_endpoints_map.py:8]: area

File: scripts/scil_image_math.py [hunk 16]
```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+"""
+Performs an operation on a list of images. The supported operations are
+listed below.
+
+This script is loading all images in memory, will often crash after a few
+hundred images.
```
Comment (mdesco) [inline scripts/scil_image_math.py:9]: Maybe we should detect this with a heuristic or send a warning if 100+ images given?

File: scripts/scil_outlier_rejection.py [hunk 17]
```diff
@@ -3,6 +3,9 @@
 """
 Clean a bundle (inliers/outliers) using hiearchical clustering.
 http://archive.ismrm.org/2015/2844.html
+
+If spurious streamlines are dense it is possible they will not be recognized as
```
Comment (mdesco) [inline scripts/scil_outlier_rejection.py:7]: dense,

File: scripts/scil_visualize_seeds.py [hunk 18]
```diff
@@ -2,8 +2,10 @@
 # -*- coding: utf-8 -*-
 
 """
-Visualize seeds where streamlines originate from
-in a tractogram
+Visualize seeds where streamlines originate from a tractogram.
```
Comment (mdesco) [inline scripts/scil_visualize_seeds.py:5]: That sound so weird... 

Visualize seeds used to generate the tractogram or bundle. i.e. when tractography was run, each streamline produced by the tracking algorithm saved its seeding point (its origin).

File: scripts/scil_image_math.py (hunk 16 above)
Comment (frheault) [reply scripts/scil_image_math.py:9]: This should be in another PR, maybe make an issue?
But nowhere in Scilpy we handle RAM or warn people about that.
Screenshotting bundle loads everything in RAM too (my similarity script too).

File: scripts/scil_bundle_volume_per_label.py (hunk 11 above)
Comment (arnaudbore) [reply scripts/scil_bundle_volume_per_label.py:6]: @frheault

File: scripts/scil_bundle_volume.py (hunk 8 above)
Comment (arnaudbore) [reply scripts/scil_bundle_volume.py]: @frheault

**Pull request merged**